controllers/bindings/serviceaccounts_test.go

```diff
@@ -413,8 +413,8 @@ func TestServiceAccountSync(t *testing.T) {
 		assert.Equal(t, "yay", storedSA.Labels["gelinkt_managed"])
 
 		// now, change the behavior to say that our SA is managed by some other target.
-		objectMarker.IsManagedByOtherImpl = func(ctx context.Context, o client.Object) (bool, error) {
```

Comment:
Is this some kind of a leftover from some time before? I see that you're adding a new method to the object market interface yet this looks like it's already been in place?

Reply:
Yeah, it's some kind of leftover that was not used.